plugins/woocommerce/tests/e2e-pw/tests/merchant/product-create-simple.spec.js

```diff
@@ -35,6 +35,8 @@ const productData = {
 	},
 };
 
+const categoryName = `cat_${ Date.now() }`;
```

Comment:
I think it's better to leave the instantiation of the category name inside the fixture, to ensure each test will have a different category name. Moving it outside means that fixture for the second and third tests will create the same category.

Reply:
I did this because using the locator became unreliable in WP 6.7.  We were using an ID locator which isn't a best practice anyhow and it differs between WP 6.6.x and WP 6.7.

The categories are cleaned up with the fixture, so while the category name is the same across all the tests, the category is being deleted/re-created with fixture creation.

Reply:
You can use `category.name`

Reply:
Huh, don't know why I thought it would be unavailable.  Done.